apps/storybook/src/Slider.stories.tsx

```diff
@@ -155,10 +177,18 @@ export const CustomTickNoTooltip: Story<SliderProps> = (args) => {
             display: 'flex',
             alignItems: 'center',
             justifyContent: 'center',
-            marginTop: '20px',
+            ...(args.orientation === 'horizontal'
```

Comment:
I would say to make this only horizontal too in order not to make it too complicated.

Reply:
I removed the option to change the orientation for `CustomTickNoTooltip`. I was also wondering, should I do the same for `` since that also involves conditionally applying styling? Below is the code:

```tsx
export const WithCustomThumb: Story<SliderProps> = (args) => {
  const thumbProps = args.thumbProps();
  thumbProps.style = {
    ...thumbProps.style,
    ...(args.orientation === 'horizontal'
      ? { transform: 'translateX(-19.2px)', top: 0 }
      : { transform: 'translate(-25%, 50%)' }),
  };
  args.thumbProps = () => thumbProps;
  return <Slider {...args} />;
};
```

Reply:
I see you did that change, so all good.